content/using_ec2_spot_instances_with_eks/spotmanagednodegroups/selecting_instance_types.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Selecting Instance Types"
+date: 2018-08-07T11:05:19-07:00
+weight: 10
+draft: false
+---
+
+[Amazon EC2 Spot Instances](https://aws.amazon.com/ec2/spot/) offer spare compute capacity available in the AWS Cloud at steep discounts compared to On-Demand prices. EC2 can interrupt Spot Instances with two minutes of notification when EC2 needs the capacity back. You can use Spot Instances for various fault-tolerant and flexible applications. Some examples are analytics, containerized workloads, high-performance computing (HPC), stateless web servers, rendering, CI/CD, and other test and development workloads.
+
+One of the best practices to successfully adopt Spot instances is to implement **Spot instance diversification** as part of your configuration. Spot instance diversification helps to procure
+capacity from multiple Spot Instance pools, both for scaling up and for replacing spot instances that may receive a spot instance termination notification. A Spot instance pool is a set of unused EC2 instances with the same instance type (for example, m5.large), operating system, Availability Zone.
```

Comment:
Try to keep it Spot Instances

Reply:
Change Spot instances, spot instances to **Spot Instances**